assignments/prototypes.js

```diff
@@ -15,13 +13,37 @@
   * dimensions (These represent the character's size in the video game)
   * destroy() // prototype method that returns: `${this.name} was removed from the game.`
 */
+function GameObject(attributes){
+  this.createdAt = attributes.createdAt,
+  this.name = attributes.name, 
+  this.dimensions = {
+    width: attributes.dimensions.width,
+    height: attributes.dimensions.height,
+    length: attributes.dimensions.length,
+  };
```

Comment:
Also a note, you could have also bound dimensions with: this.dimensions = attributes.dimensions and the width, height and length properties would have been bound.